tests/unit/src/test/scala/tests/OutlineLspSuite.scala

```diff
@@ -0,0 +1,77 @@
+package tests
+
+import scala.meta.internal.metals.codeactions.CreateNewSymbol
+import scala.meta.internal.metals.codeactions.ImportMissingSymbol
+
+class OutlineLspSuite extends BaseNonCompilingLspSuite("outline") {
+  override val scalaVersionConfig: String = ""
+  override val saveAfterChanges: Boolean = false
+  override val scala3Diagnostics = false
+
+  test("imports-for-non-compiling") {
```

Comment:
Could we open an issue to look into it later?

Reply:
Done (hopefully ok): https://github.com/scalameta/metals/issues/6593